### PR TITLE
Added documentation to run cp-web locally

### PR DESCRIPTION
**Pull request by sidrmsh**
It took me some time to figure out how to run cp-web on a local hyperledger network. So I decided it'd be useful to add documentation for it.

### CODE REVIEW

**Comment by masterDev1985** [comment]
tralalalalalalala

File: docs/use_local_hyperledger.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+#Use Local Hyperledger Network:
+
+### Creating a Local Hyperledger Network
+There is a convenient docker-compose image that will get you a network very quickly.
```
Comment (masterDev1985) [inline docs/use_local_hyperledger.md]: It isn't a docker-compose 'image'.  I'd reword this to something like, "It's easy to stand up a local hyperledger network using docker compose."

File: docs/use_local_hyperledger.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+1. Follow the docker compose [Setup Instructions](https://hub.docker.com/r/ibmblockchain/fabric-peer).
+1. Make sure your network is alive and reachable by testing the HTTP chain endpoint. To do this open your browser and browse to the peer.
+	- If you are running Windows with docker-toolbox then click http://192.168.99.100:7050/chain
```
Comment (masterDev1985) [inline docs/use_local_hyperledger.md]: I think you meant to make this one a link.

File: docs/use_local_hyperledger.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+
+### Finished
+The network is all setup. 
+
+Next we need to **copy the peer data and pass it to our demo node.js application**.
+This is done by editing the `mycreds.json` file which lives in the root of the marbles app.
+
+All we must do is edit the file with information about your network.
+If you want more details of setup options then take a look at the [SDK's documentation](https://github.com/IBM-Blockchain/ibm-blockchain-js).
```
Comment (masterDev1985) [inline docs/use_local_hyperledger.md]: Could just drop this.  We don't support `ibm-blockchain-js` anymore, and this demo doesn't use it.

Comment (dshuffma-ibm) [inline docs/use_local_hyperledger.md]: 2nd. no point in bringing people to it.

File: docs/use_local_hyperledger.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+**Do you see the "credentials" field in your json file?** 
+It should be the outter most field like in the sample above. 
+If its not there you need to add it such that `peers`, `ca` and `users` are inside `credentials`.
+
+Note that only one user (Registrar) is added to the `users` section because cp-web allows you to create new users. 
+However, you are welcome to add new users by referring to [Fabric's documentation](https://github.com/hyperledger/fabric/blob/v0.6/membersrvc/membersrvc.yaml)
+
+You can omit the field `api_port_tls` if the network does not support TLS. 
+The default docker-compose example does not support TLS. 
+Once you have edited `mycreds_docker_compose.json` you are ready to run Marbles.
```
Comment (masterDev1985) [inline docs/use_local_hyperledger.md]: You should probably change this to match the name of this demo ;)

File: README.md [hunk 5]
```diff
@@ -55,6 +61,47 @@ in the form:
...
+```
+ var registrationRequest = {
+                enrollmentID: enrollID,
+                affiliation: 'institution_a'    //change from "group1" to registrar's affiliation
+            };
+```
+6. Since the peers on the local network are listening using http, you need to make a few changes (see below).
+  - In [ws_part2.js](./utils/ws_part2.js), 
+    1. add ```var http = require('http');```
+    2. change ```https``` to ```http``` on **line 146**. It should look like this:
```
Comment (masterDev1985) [inline README.md:81]: I'm wary of including line numbers like this, as the code can change.  The correct solution here would be to build in a flag that could be set when running the demo that alters these things.  Perhaps an environment variable that you could check in `process.env`?  This is fine for now.

**Comment by masterDev1985** [comment]
@siddarthramesh You mind looking at some of the comments I made on your changes.  Just need to sort a few things out, and we'll be good to go.  Thanks for the PR!

**Comment by sidrmsh** [comment]
@masterDev1985  Thanks for the review! I made the changes on the local hyperledger doc after looking at your comments.

File: README.md (hunk 5 above)
Comment (sidrmsh) [reply README.md:81]: You're right. There should be a way to handle that in ```process.env```, but I am not sure how to do that.

**Comment by masterDev1985** [comment]
Looks good.  Thanks!
